### ISSUE TITLE
Add a few games

### ISSUE THREAD

**Issue by madeso** [madeso/ride#103]
This is great for when you need to think and do something with your hands or relax for a few minutes before getting back to work. The wx samples has a card game and a minesweeper clone. It should also be pretty straight forward to add 2048 and tetris clone.

**Comment by ConnyOnny** [comment]
Why do you want to add games to an IDE? Don't you think it would be more wise to publish games seperately?

**Comment by madeso** [comment]
Mainly 2 reasons:

1. I have a friend who plays a logic game to "relax his brain" and
2. I needed/wanted a simple way to try out how "dynamic menus" would work with 'editor-replacement's, ie text editor shouldn't interfere or clutter the dialog editor, etc.

Currently the only games that are added, are the standard wxWidgets demos so most of the work has gone to fixing the ui interface and trying to come up with a good code structure that can handle different edtiors (it's not complete but kinda works now). I'm not sure if I should add all the games, but I think a basic brainless game for your fingers like snake/2048/tetris would be a good addition so I could try out key events without the additional process/burden of adding the fake vim + prune editors that I also would like to add

So to sum it up, the main focus is not to "add games", it's to have a playground to work on requirements that other features need :)
